src/wp-includes/functions.wp-scripts.php

```diff
@@ -136,7 +136,7 @@ function wp_add_inline_script( $handle, $data, $position = 'after' ) {
  * @param array            $deps      Optional. An array of registered script handles this script depends on. Default empty array.
  * @param string|bool|null $ver       Optional. String specifying script version number, if it has one, which is added to the URL
  *                                    as a query string for cache busting purposes. If version is set to false, a version
- *                                    number is automatically added equal to current installed ClassicPress version.
+ *                                    number is automatically added equal to current installed WordPress version.
```

Comment:
This is correct for now, but before release, let's change it to ClassicPress (and change the auto-generated cache busters to match).  → https://github.com/ClassicPress/ClassicPress/issues/56

Reply:
In https://github.com/ClassicPress/ClassicPress/pull/34#discussion_r215801573 you mentioned to change it back to WordPress for now